collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -92,6 +92,7 @@ protected void onCreate(Bundle savedInstanceState) {
         currentProjectViewModel = new ViewModelProvider(this, currentProjectViewModelFactory).get(CurrentProjectViewModel.class);
         currentProjectViewModel.getCurrentProject().observe(this, project -> {
             invalidateOptionsMenu();
+            setTitle(project.getName());
```

Comment:
We need a test to drive this out. Currently, the main menu is asserted on using the app name (which is still on screen) so I think it either makes sense to add something to the Espresso tests in `projects` (probably to an existing test rather than a whole new one) or to add a test at the activity level.